release.yaml

```diff
@@ -86,5 +86,7 @@ ui-plugin-operator-crd:
 - 102.0.0+up0.2.0-rc1
 longhorn:
   - 101.2.0+up1.4.0
```

Comment:
Shouldn't we be removing 101.2.0+up1.4.0 version for longhorn and longhorn-crd, since we are adding 102.2.0+up1.4.0? cc @PennyScissors

Reply:
Yes, we should now. At the time that I created the PR, 101.2.0+up1.4.0 wasn't in the release branch yet so I have to add it here otherwise the PR verification will fail. Thank you for the reminder

Reply:
I fixed it. Thanks @prachidamle

Reply:
I thought 101 was released already, wasn't it?

Reply:
Yes, 101.2.0+up1.4.0 was released already now. So I fixed the PR to remove it. 

At the time that I created the PR, 101.2.0+up1.4.0 was not release so I had to add it here to pass the PR validation